tests/functional/spec/navigation.spec.js

```diff
@@ -200,7 +200,6 @@ describe('Navigation', function() {
     .click(SecurityCodePage.submit())
     .click(SkipInterstitialPage.no())
     .click(SkipInterstitialPage.submit())
-    .then(helpers.isSectionComplete('Payment Details')).should.eventually.be.false
```

Comment:
Really you should have changed this to true as your code changes the behaviour here so Interstitial pages no longer count towards section completeness.